docs/dictionary/function/byteOffset.lcdoc

```diff
@@ -15,17 +15,55 @@ OS: mac, windows, linux, ios, android
 Platforms: desktop, server, mobile
 
 Example:
-offset("c","abcde") -- returns 3
+byteOffset("c","abcde") -- returns 3
 
 Example:
-offset("b","abacadabra",2) -- returns 6
+byteOffset("b","abacadabra",2) -- returns 6
 
 Example:
-offset("bark","embarking") -- returns 3
+byteOffset("bark","embarking") -- returns 3
 
-References: find (command), sort container (command),
-wordOffset (function), itemOffset (function), length (function),
-lineOffset (function), offset (function), begins with (operator)
+Example:
+byteOffset(null & numberToByte(255),"End" & null & numberToByte(255)) -- returns 4
+
+Parameters:
+bytesToFind (string): a string of binary values to find
+
+
+binaryStringToSearch (string): a container with binary values
```

Comment:
can we have `<binary data>` in these two instead of `binary values` please? Also the types should be `(data)` rather than `(string)`